src/app/game-state/battle.service.ts

```diff
@@ -208,8 +208,8 @@ export class BattleService {
   }
 
   youAttack(){
+    this.characterService.characterState.accuracy = Math.min((this.troubleKills + Math.sqrt(this.characterService.characterState.attributes.speed.value)) / this.troubleKills / 2, 1)
```

Comment:
Is this going to fight back and forth with the accuracy setting in the recalculateDerivedStats function?

Reply:
Oh. Nevermind. You removed that. Any reason we set this here and not there?

Reply:
Because it's easier to use troubleKills from here rather than having to inject battle into character.